api/rest/index.php

```diff
@@ -42,7 +42,15 @@
 # For example, this will disable logic like encoding dates with XSD meta-data.
 ApiObjectFactory::$soap = false;
 
-$g_app = new \Slim\App();
+
+# Show SLIM detailed errors according to Mantis settings
+$t_config = [];
```

Comment:
Do we use `[]` instead of `array()` now?  Should we keep using array() until we drop PHP 5.x support in the future?

Reply:
Starting  with version 2.7.0 we enforce PHP >= 5.5, see https://github.com/mantisbt/mantisbt/commit/380fde11d50c22ef94210822d60b5f576c9d4a34

So using the [] array syntax should be no problem

> As of PHP 5.4 you can also use the short array syntax, which replaces array() with [].

http://php.net/manual/en/language.types.array.php

Reply:
What @atrol said. Probably need to revise coding guidelines though.
That being said, if you prefer that we stick to array() syntax, that's fine with me.

Reply:
I would stick with the old syntax at least until we update coding guidelines and upgrade mantisbt.org server.

Reply:
OK, changed.